currencies/src/main/java/com/generalbytes/batm/common/currencies/CryptoCurrency.java

```diff
@@ -90,6 +91,7 @@
     SPICE("Spice"),
     START("Startcoin"),
     SUM("Sumcoin"),
+    SUNCASH("SunCash")
```

Comment:
Missing comma

Reply:
Apologies, I've updated this accordingly.